PhuongProject/Calculator.cs

```diff
@@ -0,0 +1,82 @@
+﻿namespace PhuongProject
+{
+    public class Calculator
+    {
+        public enum Operator
+        {
+            sum,
+            subtract,
+            multiply,
+            divide
+        }
+        /// <summary>
+        /// User input and validate number
+        /// </summary>
+        /// <param name="numberName">Name of the number</param>
+        /// <returns>The input number</returns>
+        public double InputNumber(string numberName)
+        {
+            Console.Write($"Input {numberName}: ");
+            var inputNumber = Console.ReadLine();
+            double number;
+            while (!double.TryParse(inputNumber, out number))
+            {
+                Console.Write($"Invalid number. Input {numberName} again: ");
+                inputNumber = Console.ReadLine();
```

Comment:
This loop will finish until when user input a valid number. I don't know why will have to provide break condition?

Reply:
For this specific case, I would support a key (X for example) or a combination of key (Ctrl + X, for example) to help break out of the loop in case user can't enter the right data.

Reply:
When I type Ctrl + C, the program automatically exits. Seems that .Net6 already support this.